### PR TITLE
Completed css-three-column-layout -Kaitlin Lovrich

### PR DESCRIPTION
**Pull request by kaitlin-lovrich**
image

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.container {
+  max-width: 1500px;
+  margin: 0 auto;
+  flex-wrap: wrap;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: Your `.container` doesn't have `display: flex`, so this isn't doing anything, so it should be deleted

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  display: flex;
+  justify-content: center;
+  justify-content: space-between;
+}
+
+.column-full {
+  /* display: flex;
+  justify-self: center;
+  flex-wrap: wrap;
+  flex-direction: row; */
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: Delete all old code

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.column-third {
+  /* display: flex;
+  justify-content: space-between;
+  flex-direction: row; */
+
+  /* flex-wrap: wrap; */
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: Delete all old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes here Kaitlin! Another thing I noticed as well!

1. Make sure your text is centered. Compare your image to our example image. Your heading text and the text underneath the photos aren't quite aligned right

File: css-three-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            </h1>
+          </header>
+        </div>
+      </div>
+
+      <!-- Row 2 -->
+      <div class="row">
+        <div class="column-third display-flex">
+          <div class="circle">
+            <img src="./images/merge.png" alt="" />
```
Comment (shawnkost) [inline css-three-column-layout/index.html]: Needs an `alt` attribute

File: css-three-column-layout/index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+          <h2>Integrates with existing tools</h2>
+          <p>
+            Enzyme QMS integrates with your existing tools, such as JIRA, GitHub
+            and Jenkins with Enzyme
+          </p>
+        </div>
+
+        <div class="column-third display-flex">
+          <div class="circle">
+            <img src="./images/discussion.png" />
```
Comment (shawnkost) [inline css-three-column-layout/index.html]: Needs an `alt` attribute

File: css-three-column-layout/styles.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
+.container {
+  max-width: 1500px;
+  margin: 0 auto;
+  flex-wrap: wrap;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css]: Your `.container` doesn't have `display: flex`, so this isn't doing anything, so it should be deleted

**Review by shawnkost — CHANGES_REQUESTED** [review]
Few changes here Kaitlin!

**Comment by kaitlin-lovrich** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**